pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -192,8 +192,9 @@ class Build {
                             parameters: params
                     
                     // Output notification of downstream failure (the build will fail automatically)
-		    if (signJob.getResult() == 'FAILURE') {
-                        context.println "ERROR: downstream sign_build FAILED. See ${signJob.getAbsoluteUrl()}"
+					def jobResult = signJob.getResult()
+		    		if (jobResult != 'SUCCESS') {
```

Comment:
What other non-success error codes can you get?

Reply:
UNSTABLE and ABORTED